lib/league_statable.rb

```diff
@@ -36,60 +36,42 @@ def worst_offense
   end
 
   def total_home_goals 
-    total_goals = @game_teams.each_with_object({}) do |game, hash|
+    @game_teams.each_with_object({}) do |game, hash|
       hash[game.team_id] = hash[game.team_id] || [0, 0]
       hash[game.team_id] = [game.goals + hash[game.team_id][0], hash[game.team_id][1] + 1] if game.hoa == "home"
     end
   end
 
-  def highest_scoring_home_team
-    avg_goals = total_home_goals.transform_values do |value|
+  def avg_goals_results(total_goals)
```

Comment:
this is great to pass total_home/away goals methods through this. Their probably a way we could merge totaling home/away goals, but since they are two separate things, I like the readability of them being separate for now.